kolibri_instant_schools_plugin/assets/src/views/sign-up-page/index.vue

```diff
@@ -314,24 +314,19 @@
 <style lang="stylus" scoped>
 
   @require '~kolibri.styles.definitions'
+
   $iphone-5-width = 320px
   $vertical-page-margin = 100px
-  $logo-size = (1.64 * 1.125)rem
-  $logo-margin = (0.38 * $logo-size)rem
+  $logo-size = 36px
+  $logo-margin = (0.38 * $logo-size)px
```

Comment:
How did we come up with `0.38`? 0_o

Reply:
:man_shrugging:

Reply:
Such a random value haha

Reply:
What's random about 🤷‍♂️? 
😛

Reply:
@radinamatic  Do you know? :open_mouth:

Reply:
No, nvm, I was just emoji-ignorant... 🙄

Reply:
Pretty sure it's leftover from some math that was being done at some point and overwritten to to change requests from vf along the line. A lot of the original commits were made in the middle of the night on a weekend because they were critical fixes or something @christianmemije

Reply:
Yes, I bet. But while we are here, let's just change it to something like 16px? Unnecessarily smelly.

Reply:
done!